### ISSUE TITLE
pile-arxiv dataset does not have a train split

### ISSUE THREAD

**Issue by ZetangForward** [PKU-ML/LongPPL#2]
@BlackieFang Hi, I followed the download instructions to get the pile-arxiv dataset, but it doesn't include a training split. How can I use this data for training? Is it appropriate to use all of the validation and test sets as training data?

**Comment by BlackieFang** [comment]
Thanks for your question. As shown in https://github.com/PKU-ML/LongPPL/blob/main/finetune/finetune.py#L130-L134, we use the validation set of the Pile-arxiv dataset as training data. 

In fact, both the validation set and the test set of Pile-arxiv contain only ArXiv document data, which do not appear in our evaluation pipeline, so using any split as training data is reasonable.
